<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralwidget{
background-color: rgb(255, 255, 255);
background-color: rgb(66, 66, 66);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Downloads/Citibanamex_logo.svg.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>40</y>
      <width>366</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(222, 11, 19);</string>
    </property>
    <property name="text">
     <string>Registro de transferencias:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBotRegTransac">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(18, 65, 125);
font: 12pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBotCancelar">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>380</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(18, 65, 125);
color: rgb(222, 11, 19);
font: 12pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>Cancelar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>330</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Downloads/appbanamex.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>301</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(18, 65, 125);
font: 16pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Tipo de documento:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="cBox_TipoDoc">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(214, 214, 214);
font: 11pt &quot;Verdana&quot;;
color: rgb(27, 72, 130);</string>
       </property>
       <property name="currentText">
        <string/>
       </property>
       <property name="placeholderText">
        <string>-- Seleccione un documento --</string>
       </property>
       <item>
        <property name="text">
         <string>IFE/Credencial de elector</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PSS Pasaporte</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>VISA</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>301</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(18, 65, 125);
font: 16pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Motivo del giro:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="cBox_MotGiro">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(214, 214, 214);
font: 11pt &quot;Verdana&quot;;
color: rgb(27, 72, 130);</string>
       </property>
       <property name="currentText">
        <string/>
       </property>
       <property name="placeholderText">
        <string>-- Seleccione motivo de giro --</string>
       </property>
       <item>
        <property name="text">
         <string>Honorarios / Servicios técnicos</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Remesas / Giros</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Donaciones</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>100</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(18, 65, 125);
font: 16pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Número de documentos:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="linEdNumDoc">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(225, 225, 225);
color: rgb(18, 65, 125);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>16</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(18, 65, 125);
font: 16pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Monto:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="linEdMonto">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(225, 225, 225);
color: rgb(18, 65, 125);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>248</width>
      <height>60</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QCheckBox" name="chekBoxTransInter">
       <property name="styleSheet">
        <string notr="true">color: rgb(18, 65, 125);
font: 12pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Transferencia internacional</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="chekBoxDolares">
       <property name="styleSheet">
        <string notr="true">color: rgb(18, 65, 125);
font: 12pt &quot;Verdana&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Dólares</string>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Topicos/.venv/Lib/site-packages/PySide6/interfaces/logocitibanamex-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>320</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Topicos/.venv/Lib/site-packages/PySide6/interfaces/logopeque-removebg-preview.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>833</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>cBox_TipoDoc</tabstop>
  <tabstop>cBox_MotGiro</tabstop>
  <tabstop>linEdNumDoc</tabstop>
  <tabstop>linEdMonto</tabstop>
  <tabstop>chekBoxTransInter</tabstop>
  <tabstop>chekBoxDolares</tabstop>
  <tabstop>pushBotRegTransac</tabstop>
  <tabstop>pushBotCancelar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
